### DISCUSSION TITLE
Allow relation manager to be collapsible

### DISCUSSION THREAD

**Discussion by underdpt** [filamentphp/filament#9267] [Ideas]
Hello,

It would be useful to be able to make a relation manager collapsible (and collapsed by default). This would be useful when the relation manager information is optional, so it wouldn't take much screen space and would not load the data until the relation manager is expanded.

**Comment by dissto** [comment]
Probably not exactly what you want but if all you want is have more space you could try to combine the relation managers into tabs.

https://filamentphp.com/docs/2.x/admin/resources/relation-managers#moving-the-resource-form-to-tabs

**Reply by underdpt** [reply]
Hi @dissto 

Yes, I know that. But there's always a relation manager shown, and that's what I would like to change, so you can view that there are related data, but don't show or load that data until the user needs it.

**Reply by zeeshantariq08** [reply]
hi @underdpt you can use con conditionally to show the relation manager 
```
        use Illuminate\Database\Eloquent\Model;
         
        public static function canViewForRecord(Model $ownerRecord, string $pageClass): bool
        {
            return $ownerRecord->status === Status::Draft;
        }
```

**Comment by adam-martinez** [comment]
Collapsible tables would be great all around!

**Comment by leoblanski** [comment]
It would be very useful !!

**Comment by luishecastilho** [comment]
I really need that

**Comment by geraistec** [comment]
Should be standard

**Comment by mcnabsystems** [comment]


**Reply by geraistec** [reply]
I believe that Filament's philosophy is exactly to do away with complication.

A simple "->collapsibleForm()" would do the trick.
